CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Versions
 ========
 
+2018-08-DEV
+-----------
+
+* Add `basic-build`, `libffi-dev` and `openldap-dev` packages in Python images
```

Comment:
Can you move these 2 lines in `2018-07-DEV` and rename it to `2018-08-DEV` please?

Reply:
Changelog updated